server/embedded/src/org/labkey/embedded/LabKeyServer.java

```diff
@@ -182,7 +186,7 @@ protected TomcatWebServer getTomcatWebServer(Tomcat tomcat)
                     {
                         if (contextProperties.getContextPath() != null && !contextProperties.getContextPath().isEmpty() && !contextProperties.getContextPath().equals("/"))
                         {
-                            throw new IllegalArgumentException("contextPath.legacyContextPath is only intended for use when deploying the LabKey application to the root context path. Please update application.properties.");
+                            throw new ConfigException("contextPath.legacyContextPath is only intended for use when deploying the LabKey application to the root context path. Please update application.properties.");
```

Comment:
Are we catching this in the appropriate locations since this isn't a subclass of the old Exception type?

Reply:
Nope! These are considered fatal problems and are thrown all the way to the top, killing the startup attempt.